lib/smart_answer_flows/marriage-abroad.rb

```diff
@@ -361,7 +361,6 @@ def define
       outcome :outcome_opposite_sex_marriage_in_south_korea
       outcome :outcome_opposite_sex_marriage_in_slovakia
       outcome :outcome_opposite_sex_marriage_in_thailand
-      outcome :outcome_opposite_sex_marriage_in_turkey
```

Comment:
This line should probably be removed in the following commit (7506402f8cfd9dfef3e6ce7b26f9a8c60be7eac1) where the extraneous `turkey` code is removed.